src/rules/dollar-variable-colon-newline-after/index.js

```diff
@@ -0,0 +1,62 @@
+import {
+  declarationValueIndex,
+  namespace,
+  whitespaceChecker,
+} from "../../utils"
+import { utils } from "stylelint"
+
+export const ruleName = namespace("dollar-variable-colon-newline-after")
+
+export const messages = utils.ruleMessages(ruleName, {
+  expectedAfter: () => "Expected newline after \":\"",
+  rejectedAfter: () => "Unexpected whitespace after \":\"",
```

Comment:
Indeed it seems so. I was initially relying on `declaration-colon-newline-after` rule from stylelint, so I thought it could be. Will remove the message and correct the rule's description.